koha-tmpl/intranet-tmpl/prog/en/includes/js-patron-format-address.inc

```diff
@@ -11,22 +11,27 @@
             return map;
         }, {});
 
-        function member_display_address_style_us(data, line_break) {
+        function member_display_address_style_us(data, line_break, include_li) {
```

Comment:
It would be nice to have some jsdoc here for the meaning of 'line_break' vs 'include_li'

Reply:
Correct me if I'm wrong but this change is not part of this work, this is coming from this patch
https://github.com/PTFS-Europe/koha/commit/dbd0117a494e516265a2598a8f6b60f896f57c79

Is it not?